sdk/core/core-client-rest/src/common.ts

```diff
@@ -274,5 +274,13 @@ export interface ErrorModel {
   /** An array of details about specific errors that led to this reported error. */
   details: Array<ErrorModel>;
   /** An object containing more specific information than the current object about the error. */
-  cause?: ErrorModel;
+  innererror?: InnerError;
```

Comment:
should it be `innerError`?

Reply:
it comes from cadl https://github.com/Azure/cadl-azure/blob/main/packages/cadl-azure-core/lib/foundations.cadl#L112 here. as RLC doesn't have a mapper and serializer, deserializer, I think RLC should keep consistent with CADL.